src/transformers/dependency_versions_check.py

```diff
@@ -0,0 +1,14 @@
+from .dependency_versions_table import deps
+from .file_utils import is_tokenizers_available
+from .utils.versions import require_version_core
+
+
+# define which module versions we always want to check at run time (usually the ones defined in `install_requires` in setup.py)
+pkgs_to_check_at_runtime = "python tokenizers tqdm regex numpy".split()
+for pkg in pkgs_to_check_at_runtime:
+    if pkg in deps:
+        if pkg == "tokenizers" and not is_tokenizers_available():
```

Comment:
I'm playing around with the functionalities a  bit and I noticed that if someone has *e.g.* `tqdm==4.16` installed instead of getting the error:

```
pkg_resources.VersionConflict: tqdm>=4.27 is required for a normal functioning of this module, but found
tqdm==4.16.0.
```

one gets the normal error:
```
ModuleNotFoundError: No module named 'tqdm.auto'
```

=> the reason is that tqdm is already required when importing `is_tokenizers_available()`. 
Could we maybe do the following to solve this:

a) put `tokenizers` at the very last place in the pkgs list:
```
pkgs_to_check_at_runtime = "python tqdm regex numpy tokenizers".split()
```
b) import `is_tokenizers_available()` dynamically under `if pkg == "tokenizers":` so that the correct error message is shown for `tqdm`?

Reply:
good catch, @patrickvonplaten - both your suggestions were needed. Applied.